web/admin/src/Admin.js

```diff
@@ -130,7 +156,17 @@ export default class Admin extends PureComponent {
   }
 
   addQuestion = (sessionId) => {
-    this.questionsRef().push({sessionId, order: this.questionsForCurrentSession().length, text: '', options: ['','','',''], correctIndex: 0})
+    const questions = this.questionsForCurrentSession()
+    if (questions.length) {
+      const checkQ = questions[questions.length-1]
+      if (checkQ.text && checkQ.options[0]) this.questionsRef().push({sessionId, order: this.questionsForCurrentSession().length, text: '', options: ['','','',''], correctIndex: 0})
```

Comment:
An `if/else` should at least indent the 2 branches, and preferably have braces around them for readability and preventing future logic errors.  It only adds 2 lines for the trailing braces.